presto-main/src/main/java/io/prestosql/memory/NodeMemoryConfig.java

```diff
@@ -51,15 +51,15 @@ public NodeMemoryConfig setMaxQueryMemoryPerNode(DataSize maxQueryMemoryPerNode)
         return this;
     }
 
-    public boolean isReservedPoolEnabled()
+    public boolean isReservedPoolDisabled()
     {
-        return isReservedPoolEnabled;
+        return isReservedPoolDisabled;
     }
 
-    @Config("experimental.reserved-pool-enabled")
-    public NodeMemoryConfig setReservedPoolEnabled(boolean reservedPoolEnabled)
+    @Config("experimental.reserved-pool-disabled")
```

Comment:
We may want to add a legacy config with the old name to ease transition for anyone that has that option set. If I recall correctly, airlift will report an error if both the old and new properties are specified simultaneously.

Reply:
AC

indeed it does
```
1) Error: Configuration property 'experimental.reserved-pool-enabled' (=false) conflicts with property 'experimental.reserved-pool-disabled' (=true)
```